dist/index.html

```diff
@@ -1 +1,22 @@
-<!doctype html><html lang="en"><head><meta charset="UTF-8"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="viewport" content="width=device-width,initial-scale=1"><title><to-do-list></to-do-list></title><script defer="defer" src="main.js"></script></head><body><main><h1>To-dos</h1><table id="tasks"></table></main></body></html>
\ No newline at end of file
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js" integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM" crossorigin="anonymous"></script>
+    <title>To-dos-List</title>
+<script defer src="main.js"></script></head>
+<body>
+    <main class="m-4 w-70">
+        <div class="border border-secondary d-flex flex-column align-items-center justify-content-center rounded-3">
+            <h2 class="py-1 px-2 mx-2">To-Dos</h2>
+              <ul id="tasks" class="listClass d-flex w-50 flex-column" class = "drag-sort-enable"></ul>
```

Comment:
Let's try to follow good [HTML/CSS practice](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md):
`class = "drag-sort-enable"` Use space-less equal signs, like this `class="drag-sort-enable"`